### PR TITLE
 qtivi-mopidy-plugin bump and minimal image build fix

### PR DESCRIPTION
**Pull request by sashko**

### CODE REVIEW

File: recipes-multimedia/qtivi-mopidy-plugin/qtivi-mopidy-plugin_git.bb [hunk 1]
```diff
@@ -12,8 +12,6 @@ SRCREV = "18198a616501bca072689a31c1581c879605da68"
 
 S = "${WORKDIR}/git"
 
-inherit qmake5
```
Comment (TSAnsari) [inline recipes-multimedia/qtivi-mopidy-plugin/qtivi-mopidy-plugin_git.bb]: Does removing this affect neptune images?

Comment (sashko) [reply recipes-multimedia/qtivi-mopidy-plugin/qtivi-mopidy-plugin_git.bb]: No, it builds fine.

**Comment by JEderonn** [comment]
Consider to move qtivi-mopidy-plugin from core-image-pelux-qtauto.bbclass to layers/b2qt/recipes-core/images/core-image-pelux-qtauto-neptune<-dev>.bb.

And consider to move mopidy from core-image-pelux-minimal.bbclass to recipes-core/images/core-image-pelux-minimal<-dev>.bb.

Main reason for this is to not have Mopidy in all PELUX-based images. It does not make sense to have mopidy on a non-IVI system like e.g. telematics.

An option would be to use DISTRO_FEATURES to enable mopidy, and to define mopidy in the relevant bb-files.

**Comment by sashko** [comment]
> Consider to move qtivi-mopidy-plugin from core-image-pelux-qtauto.bbclass to layers/b2qt/recipes-core/images/core-image-pelux-qtauto-neptune<-dev>.bb.
> 
> And consider to move mopidy from core-image-pelux-minimal.bbclass to recipes-core/images/core-image-pelux-minimal<-dev>.bb.
> 
> Main reason for this is to not have Mopidy in all PELUX-based images. It does not make sense to have mopidy on a non-IVI system like e.g. telematics.
> 
> An option would be to use DISTRO_FEATURES to enable mopidy, and to define mopidy in the relevant bb-files.

Having `DISTRO_FEATURES` might be a good thing to do, but I do not understand why you want Mopidy to be moved to -dev images.

**Comment by JEderonn** [comment]
> > Consider to move qtivi-mopidy-plugin from core-image-pelux-qtauto.bbclass to layers/b2qt/recipes-core/images/core-image-pelux-qtauto-neptune<-dev>.bb.
> > And consider to move mopidy from core-image-pelux-minimal.bbclass to recipes-core/images/core-image-pelux-minimal<-dev>.bb.
> > Main reason for this is to not have Mopidy in all PELUX-based images. It does not make sense to have mopidy on a non-IVI system like e.g. telematics.
> > An option would be to use DISTRO_FEATURES to enable mopidy, and to define mopidy in the relevant bb-files.
> 
> Having `DISTRO_FEATURES` might be a good thing to do, but I do not understand why you want Mopidy to be moved to -dev images.

Well, this is what happens today since the dev-images inherits core-image-pelux.bbclass where it currently is included by default. 

Issue OAI1209-565 is a planned story which purpose is to define which image variants we shall have and what they shall include. For now, let's do what is required to make images to build.

**Pull request merged**